st2actions/bin/runners.sh

```diff
@@ -28,7 +28,7 @@ elif [ -z "$sv" ] && ( /sbin/start 2>&1 | grep -q "missing job name" ); then
   sv=upstart
   svbin=$UPSTARTCTL
 else
-  # Old debians, redhats and centos, amazon etc
+  # Old debians, amazon etc
   sv=sysv
```

Comment:
Good that you found this file!

`EL6` was the last platform with `sysv` init system which relies on `/etc/init.d/` service startup files. All OS systems we support right now have `systemd` only.

Please try to cleanup this file based on that information, I'll help where I can.